build/cpython/Makefile

```diff
@@ -105,6 +105,7 @@ pyenv: /opt/redislabs $(CPYTHON_PREFIX) $(PYENV_DIR)
 
 $(CPYTHON_PREFIX): /opt/redislabs
 	$(SHOW)make -C $(BUILD_DIR) install 2>&1 >$(BUILD_DIR)/python-install.log
+	$(SHOW)objcopy --localize-symbol PyGILState_Ensure --localize-symbol PyGILState_Release $(BINDIR)/libpython3.7m.a
```

Comment:
I tried finding a variable that has the lib names instead of hardcoding. It exists in Makefile.defs and there's already a commented-out include for it at the top of this file.